test/functional/capsule/content/remove_lifecycle_environment_test.rb

```diff
@@ -15,14 +16,14 @@ class Content
         it 'allows minimal options' do
           expect_lifecycle_environment_remove('id' => 1, 'environment_id' => 2)
 
-          run_cmd(%w(capsule content remove-lifecycle-environment --id 1 --environment-id 2))
+          run_cmd(%w(capsule content remove-lifecycle-environment --id 1 --lifecycle-environment-id 2))
         end
 
         it 'resolves environment ID from name' do
           expect_lifecycle_environment_search('3', 'env2', 2)
           expect_lifecycle_environment_remove('id' => 1, 'environment_id' => 2)
 
-          run_cmd(%w(capsule content remove-lifecycle-environment --id 1 --environment env2
+          run_cmd(%w(capsule content remove-lifecycle-environment --id 1 --lifecycle-environment env2
```

Comment:
If you're dropping support for `--environment` in tests, maybe it's time to actually drop it from the commands?

Reply:
@ofedoren I was just trying to fix the tests, do you think we should keep it?